vllm/model_executor/layers/rejection_sampler.py

```diff
@@ -7,6 +7,11 @@
 from vllm.model_executor.layers.spec_decode_base_sampler import (
     SpecDecodeStochasticBaseSampler)
 
+try:
+    from flashinfer.sampling import chain_speculative_sampling
+except ImportError:
```

Comment:
nit: can add a comment explaining the motivation for a separate kernel? e.g. a blurb about how flash infer is faster